### ISSUE TITLE
What does this button do?

### ISSUE THREAD

**Issue by hoverkw** [CMCSHE201/CheapShark_API_App#56]
I'm not aware of the purpose of this button. Is this some kind of the artifact left during development?

![7](https://user-images.githubusercontent.com/62996643/152641210-a0fd91fb-a5aa-4979-89f2-387165a21f52.png)

**Comment by CMCSHE201** [comment]
That will be the button to minimize the navigation bar so the user can view the pages without obstruction. The functionality is still in progress but will be added soon.

**Comment by jc-gc** [comment]
Non-functional button left in design, will be rectified shortly in an upcoming release.

**Comment by hoverkw** [comment]
Thanks, appreciate your feedback.
